New images for November 2024 Spotlight

amada-ech opened the pull request:
here they are!

## What are you changing in this pull request and why?
<!--
I need to launch the Nov 2024 Spotlight. In order to do this I need to upload headshots, and create new entires.

Checklist before the PR is merged:
- [ ] Make sure the new entries all have a Community Award Recipient 2024 tag (needs to be created ASAP by web team kindly please!
- [ ] I need to upload a new headshot for 1 of the recipients which is a collage of 5 images...I am waiting on design to help me create that ASAP
- [ ] Need technical review to make sure everything looks good from Web Team and /or Docs team

-->

## Checklist
- [ x] I have reviewed the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [ ] The topic I'm writing about is for specific dbt version(s) and I have versioned it according to the [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and/or [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content) guidelines.
- [x ] I have added checklist item(s) to this list for anything anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."
- [ ] The content in this PR requires a dbt release note, so I added one to the [release notes page](https://docs.getdbt.com/docs/dbt-versions/dbt-cloud-release-notes).
<!--
PRE-RELEASE VERSION OF dbt (if so, uncomment):
- [ ] Add a note to the prerelease version [Migration Guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/docs/dbt-versions/core-upgrade)
-->
<!-- 
ADDING OR REMOVING PAGES (if so, uncomment):
- [ ] Add/remove page in `website/sidebars.js`
- [ ] Provide a unique filename for new pages
- [ ] Add an entry for deleted pages in `website/vercel.json`
- [ ] Run link testing locally with `npm run build` to update the links that point to deleted pages
-->

amada-ech committed:
New images for November 2024 Spotlight

here they are!

amada-ech commented:
I just realized I maybe shouldnt have titled this "New images for November 2024 Spotlight"... if the spotlight entries themselves will fall under the same PR, then this project is more than just images, its the 8 spotlight entries themselves too FYI!

amada-ech committed:
Create jenna-jordan.md

New card for Jenna!

Did I do all the links right?

I wanted all of them to open in a new tab. I know for the Description I had to use Markdown links and I know I do notttt have to use them in the other fields, but I did anyway because that's the only way I know how to make the link open in a new tab

amada-ech committed:
Update jenna-jordan.md

set 
dateCreated to 2024-11-01
and added a line
communityAwardYear:2024

matthewshaver commented on website/docs/community/spotlight/jenna-jordan.md:
```diff
@@ -0,0 +1,37 @@
+id: jenna-jordan
+title: Jenna Jordan
+description: |
+  I am a Senior Data Management Consultant with Analytics8, where I advise clients on dbt best practices (especially regarding dbt Mesh and the various shifts in governance and strategy that come with it). My experiences working within a dbt Mesh architecture and all of the difficulties organizations could run into with such a major paradigm shift inspired my peer exchange (role-playing/simulatio ...
```
Syntax might be breaking Vercel build
```suggestion
  I am a Senior Data Management Consultant with Analytics8, where I advise clients on dbt best practices (especially regarding dbt Mesh and the various shifts in governance and strategy that come with it). My experiences working within a dbt Mesh architecture and all of the difficulties organizations could run into with such a major paradigm shift inspired my peer exchange (role-playing/simulation game) at <a href="https://coalesce.getdbt.com/agenda" rel="noopener noreferrer" target="_blank">Coalesce 2024</a>: Governance co-lab: We the people, in order to govern data, do establish processes. I also experimented with bringing role-playing scenarios to data problems at the September 2024 <a href="https://www.meetup.com/chicago-dbt-meetup/" rel="noopener noreferrer" target="_blank">Chicago dbt Meetup</a>, hosted by Analytics8. I occasionally write long blog posts on my website, if you're up for the read.
```

matthewshaver committed:
Update website/docs/community/spotlight/jenna-jordan.md

amada-ech committed:
Create christophe-oudoar.md

new page!

matthewshaver commented on website/docs/community/spotlight/jenna-jordan.md:
```diff
@@ -0,0 +1,37 @@
+id: jenna-jordan
+title: Jenna Jordan
+description: |
+  I am a Senior Data Management Consultant with Analytics8, where I advise clients on dbt best practices (especially regarding dbt Mesh and the various shifts in governance and strategy that come with it). My experiences working within a dbt Mesh architecture and all of the difficulties organizations could run into with such a major paradigm shift inspired my peer exchange (role-playing/simulatio ...
```
Wasn't the problem
```suggestion
  I am a Senior Data Management Consultant with Analytics8, where I advise clients on dbt best practices (especially regarding dbt Mesh and the various shifts in governance and strategy that come with it). My experiences working within a dbt Mesh architecture and all of the difficulties organizations could run into with such a major paradigm shift inspired my peer exchange (role-playing/simulation game) at <a href="https://coalesce.getdbt.com/agenda" rel="noopener noreferrer" target="_blank">Coalesce 2024</a>: "Governance co-lab: We the people, in order to govern data, do establish processes." I also experimented with bringing role-playing scenarios to data problems at the September 2024 <a href="https://www.meetup.com/chicago-dbt-meetup/" rel="noopener noreferrer" target="_blank">Chicago dbt Meetup</a>, hosted by Analytics8. I occasionally write long blog posts on my website, if you're up for the read.
```

matthewshaver committed:
Update website/docs/community/spotlight/jenna-jordan.md

matthewshaver commented on website/docs/community/spotlight/jenna-jordan.md:
```diff
@@ -0,0 +1,37 @@
+
+## When did you join the dbt community and in what way has it impacted your career?
+
+My dbt learning journey kicked off with the CoRise (now Uplimit) course <a href="https://uplimit.com/course/analytics-engineering-with-dbt/“ rel="noopener noreferrer" target="_blank">Analytics Engineering with dbt</a>, with Emily Hawkins and Jake Hannan, in February 2022 – less than a month after starting as a data engineer with the City of Boston Analytics Team. About a year later, I spearheaded ...
```
```suggestion
My dbt learning journey kicked off with the CoRise (now Uplimit) course <a href="https://uplimit.com/course/analytics-engineering-with-dbt/" rel="noopener noreferrer" target="_blank">Analytics Engineering with dbt</a>, with Emily Hawkins and Jake Hannan, in February 2022 – less than a month after starting as a data engineer with the City of Boston Analytics Team. About a year later, I spearheaded the adoption of dbt at the City and got to build the project and associated architecture from scratch – which is probably the best learning experience you could ask for! I saw the value dbt could bring to improving data management processes at the City, and I knew there were other cities and local governments that could benefit from dbt as well, which motivated me to find my fellow co-speakers Ian Rose and Laurie Merrell to give a talk at Coalesce 2023 called <a href="https://www.youtube.com/watch?v=6aX7tAfMmIM&“ rel="noopener noreferrer" target="_blank">“From Coast to Coast: Implementing dbt in the public sector”.</a> As a part of our goal to identify and cultivate a community of dbt practitioners in the public (and adjacent) sectors, we also started the dbt Community Slack channel <a href="https://getdbt.slack.com/archives/C05MNU6QB5L/“ rel="noopener noreferrer" target="_blank">#industry-public-sector</a>. That experience allowed me to continue to grow my career and find my current role - as well as connect with so many amazing data folks!
```

matthewshaver committed:
Update website/docs/community/spotlight/jenna-jordan.md

matthewshaver commented on website/docs/community/spotlight/christophe-oudoar.md:
```diff
@@ -0,0 +1,35 @@
+    link: https://x.com/Kayrnt
+  - name: LinkedIn
+    link: https://www.linkedin.com/in/christopheoudar/
+ - name: Substack
```
```suggestion
  - name: Substack
```

matthewshaver committed:
Update website/docs/community/spotlight/christophe-oudoar.md

matthewshaver commented on website/docs/community/spotlight/jenna-jordan.md:
```diff
@@ -0,0 +1,37 @@
+
+## When did you join the dbt community and in what way has it impacted your career?
+
+My dbt learning journey kicked off with the CoRise (now Uplimit) course <a href="https://uplimit.com/course/analytics-engineering-with-dbt/" rel="noopener noreferrer" target="_blank">Analytics Engineering with dbt</a>, with Emily Hawkins and Jake Hannan, in February 2022 – less than a month after starting as a data engineer with the City of Boston Analytics Team. About a year later, I spearheaded ...
```
```suggestion
My dbt learning journey kicked off with the CoRise (now Uplimit) course <a href="https://uplimit.com/course/analytics-engineering-with-dbt/" rel="noopener noreferrer" target="_blank">Analytics Engineering with dbt</a>, with Emily Hawkins and Jake Hannan, in February 2022 – less than a month after starting as a data engineer with the City of Boston Analytics Team. About a year later, I spearheaded the adoption of dbt at the City and got to build the project and associated architecture from scratch – which is probably the best learning experience you could ask for! I saw the value dbt could bring to improving data management processes at the City, and I knew there were other cities and local governments that could benefit from dbt as well, which motivated me to find my fellow co-speakers Ian Rose and Laurie Merrell to give a talk at Coalesce 2023 called <a href="https://www.youtube.com/watch?v=6aX7tAfMmIM&" rel="noopener noreferrer" target="_blank">“From Coast to Coast: Implementing dbt in the public sector”.</a> As a part of our goal to identify and cultivate a community of dbt practitioners in the public (and adjacent) sectors, we also started the dbt Community Slack channel <a href="https://getdbt.slack.com/archives/C05MNU6QB5L/“ rel="noopener noreferrer" target="_blank">#industry-public-sector</a>. That experience allowed me to continue to grow my career and find my current role - as well as connect with so many amazing data folks!
```

matthewshaver committed:
Update website/docs/community/spotlight/jenna-jordan.md

matthewshaver commented on website/docs/community/spotlight/jenna-jordan.md:
```diff
@@ -0,0 +1,37 @@
+
+## When did you join the dbt community and in what way has it impacted your career?
+
+My dbt learning journey kicked off with the CoRise (now Uplimit) course <a href="https://uplimit.com/course/analytics-engineering-with-dbt/" rel="noopener noreferrer" target="_blank">Analytics Engineering with dbt</a>, with Emily Hawkins and Jake Hannan, in February 2022 – less than a month after starting as a data engineer with the City of Boston Analytics Team. About a year later, I spearheaded ...
```
```suggestion
My dbt learning journey kicked off with the CoRise (now Uplimit) course <a href="https://uplimit.com/course/analytics-engineering-with-dbt/" rel="noopener noreferrer" target="_blank">Analytics Engineering with dbt</a>, with Emily Hawkins and Jake Hannan, in February 2022 – less than a month after starting as a data engineer with the City of Boston Analytics Team. About a year later, I spearheaded the adoption of dbt at the City and got to build the project and associated architecture from scratch – which is probably the best learning experience you could ask for! I saw the value dbt could bring to improving data management processes at the City, and I knew there were other cities and local governments that could benefit from dbt as well, which motivated me to find my fellow co-speakers Ian Rose and Laurie Merrell to give a talk at Coalesce 2023 called <a href="https://www.youtube.com/watch?v=6aX7tAfMmIM&" rel="noopener noreferrer" target="_blank">"From Coast to Coast: Implementing dbt in the public sector".</a> As a part of our goal to identify and cultivate a community of dbt practitioners in the public (and adjacent) sectors, we also started the dbt Community Slack channel <a href="https://getdbt.slack.com/archives/C05MNU6QB5L/" rel="noopener noreferrer" target="_blank">#industry-public-sector</a>. That experience allowed me to continue to grow my career and find my current role - as well as connect with so many amazing data folks!
```

matthewshaver committed:
Update website/docs/community/spotlight/jenna-jordan.md

matthewshaver committed:
Merge branch 'current' into amada-ech-nov2024-spotlight

amada-ech committed:
Create ruth-onyekwe.md

New card for Ruth!

amada-ech committed:
Update jenna-jordan.md

Just deleting this
organization: ""

amada-ech committed:
Rename christophe-oudoar.md to christophe-oudar.md

Rename christophe-oudoar.md to christophe-oudar.md

amada-ech committed:
Update meagan-palmer.md

-Updated date created since we are posting her again
-Updated community award to true, and added the community award 2024 tag
-Edited/added 2 links in HTML format (supporting opening in a new tab)

amada-ech committed:
Update fabiyi-opeyemi.md

Updated date created
Updated award to 2024

amada-ech committed:
Update fabiyi-opeyemi.md

No changes realized when I updated the date to 2024-11-02 and the jobTitle.... I did it in the "current" branch... I didn't do it imy new "amada-ech-nov2024-spotlight"PR which I'm supposed to be using, so I'm worried it won't update when we go live on Monday

amada-ech committed:
Update bruno-de-lima.md

-updated headshot image
-updated job title
-updated dateCreated to 2024 instead of 2023
-updated award from 2023 to 2024
-updated "About/description" text
-updated the answer to When did you join the dbt community and in what way has it impacted your career?

amada-ech committed:
Create mike-stanley.md

Mike!

amada-ech committed:
athena maintainers headshot - 2nd attempt

my 2nd attempt at doing this

matthewshaver committed:
Merge branch 'current' into amada-ech-nov2024-spotlight

amada-ech committed:
Create original-dbt-athena-maintainers.md

runleonarun committed:
fixing paragraph
